presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMergeStatisticalDigestFunction.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation;
+
+import com.facebook.presto.spi.Page;
+import com.facebook.presto.spi.block.Block;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.function.BiFunction;
+
+import static com.facebook.presto.operator.aggregation.AggregationTestUtils.assertAggregation;
+
+public abstract class TestMergeStatisticalDigestFunction
+        extends AbstractTestAggregationFunction
+{
+    protected abstract BiFunction<Object, Object, Boolean> getEquality();
+
+    @Override
+    public abstract Block[] getSequenceBlocks(int start, int length);
```

Comment:
Can this be protected

Reply:
It's public in `AbstractTestAggregationFunction`, which is the parent class.